.github/workflows/clang_sanitizers.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - "development"
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+    types: [opened, synchronize, reopened, ready_for_review, auto_merge_enabled]
```

Comment:
@EZoni If I open a PR in ready state and then enable `auto_merge_enabled`, does it run the CI _twice_?

Reply:
@ax3l 

As discussed offline, I suppose yes, though I'm not 100% sure.

I removed the trigger, we'll close and reopen to trigger GitHub Actions workflows for now. 

The PR is updated with the cron schedule fixes and ready to review/merge.